### PR TITLE
Feat - Update chance library definitions

### PR DESCRIPTION
**Pull request by cafesanu**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<http://chancejs.com/>>
- [x] Increase the version number in the header if appropriate.

---
This change is [Reviewable](https://reviewable.io/reviews/definitelytyped/definitelytyped/23771)

### CODE REVIEW

**Comment by cafesanu** [comment]
@cbowdon 
@brikou

**Comment by cbowdon** [comment]
I'm no longer an active chance user and cannot advise. Please feel free to remove my credit from chance/index.d.ts.

File: types/chance/index.d.ts [hunk 1]
```diff
@@ -225,13 +236,16 @@ declare namespace Chance {
         name: string;
         abbreviation: string;
     }
-}
 
-// window.chance
-declare var chance: Chance.Chance;
-declare var Chance: Chance.ChanceStatic;
```
Comment (cafesanu) [inline types/chance/index.d.ts]: these two lines assume that chance is installed globally for everyone. Deleting it as typescript thing is fine to use `chance` even if it is not global

File: types/chance/index.d.ts [hunk 2]
```diff
@@ -225,13 +236,17 @@ declare namespace Chance {
...
```
Comment (cafesanu) [inline types/chance/index.d.ts]: These two lines above assume everyone has chance installed globally. Deleting them as Typescript assumes this is good even when chance is not global

**Pull request merged**